xbmc/network/DNSNameCache.cpp

```diff
@@ -55,6 +55,7 @@ bool CDNSNameCache::Lookup(const std::string& strHostName, std::string& strIpAdd
     return true;
 
 #ifndef TARGET_WINDOWS
+#ifdef HAS_SMBCLIENT
```

Comment:
please check how the correct define is named
it should replace the `#ifndef TARGET_WINDOWS`